.github/workflows/documentation-ci.yml

```
@@ -4,9 +4,15 @@ on:
     branches: ["main"]
     paths: ["docs/sources/**"]
   workflow_dispatch:
+
+permissions: {}
+
 jobs:
   vale:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

To upload the SARIF file, I believe we also need:

```suggestion
      pull-requests: write
      security-events: write
```